README.md

```diff
@@ -15,5 +15,14 @@ Cloud CLI.
 We welcome contributions to Config Sync from the community. Take a look at our
 [contribution guide] to get started.
 
+## Community, discussion and support
+You can reach the maintainers of this project at:
+
+* [github.com/GoogleContainerTools/kpt-config-sync]
```

Comment:
Do we want to emphasize the discussion groups?

Reply:
Added the discussion group to the top of the list